### ISSUE TITLE
Additional Exercises section

### ISSUE THREAD

**Issue by wilkus27** [Connexions/cnx-recipes#658]
On katalyst01 (https://katalyst01.cnx.org/contents/LnN76Opl@14.4:s6cgg-6Q@14.4/Exercises) is additional empty section **Exercises**. 

![empty_ex](https://user-images.githubusercontent.com/40228854/46613938-d483ec00-cb14-11e8-8b56-5fa633c8b3fe.JPG)

In PDF at end of the chapter is just one module **For Further Exploration**.

**Comment by Adrianltd** [comment]
@helenemccarron @openstaxalina Because the `Exercises` have `childPages`, the recipes create that container and headers even if there's nothing inside.
I think that it can be solved either by content managers, I guess that it can be done by removing 
`<cnx-pi data-type="cnx.eoc">class="review-questions" title="Review Questions"</cnx-pi>
<cnx-pi data-type="cnx.eoc">class="thought-questions" title="Thought Questions"</cnx-pi>
<cnx-pi data-type="cnx.eoc">class="figuring-for-yourself" title="Figuring for Yourself"</cnx-pi>`
 from the 1st chapter or we would have to change recipes.
And then in recipes we would have to either move those empty exercises to trash or create some kind of `if` statement that would check if the content exists before it creates that page.

**Comment by openstaxalina** [comment]
@Adrianltd thank you for the analysis.  Yes, chapter 1 is different from other chapters and does not have summaries, any type of exercise, or a collaborative group activity.  The only collating material it has is the For Further Exploration, as Natalia noted.

@lwtchu, @helenemccarron, please review this suggested content change at the next meeting.

**Comment by helenemccarron** [comment]
@Adrianltd I think this might be recipe issue. The empty section should be ignored and not appear. I think it might be related to the order in which the recipes are creating the titles but I would have to look into it a bit more in depth.

**Comment by openstaxalina** [comment]
The recipes do not use the collation instructions, so removing them from the content will not resolve this issue.  Further code investigation needed.

**Comment by helenemccarron** [comment]
This really only addresses issue #658 but impacts all books that contain compound EOC sections (Astronomy, Dev-Math and University Physics). 

![2018-11-19_15-37-58](https://user-images.githubusercontent.com/8514591/48736519-1bcfc300-ec11-11e8-9a24-3e6649272d59.png)


~TODO: figure out what happened to the key terms in U-Physics~

For the indexed terms in the EOC section to behave properly, I had to move `modify_finishPrepingIndexTerms` to its own pass, thus bumping all subsequent passes. 

All recipes were modified but without any change to the output. I diffed the before and after html for books other than astronomy, dev-math and university physics and both files are identical.

University Physics vol. 1 returned one change, which I believe is an improvement:
![2018-11-19_10-51-34](https://user-images.githubusercontent.com/8514591/48736379-b1b71e00-ec10-11e8-9b44-44317417d6ea.png)

**Comment by wilkus27** [comment]
Verified on katalyst01.

In Astronomy it displays properly. There is no additional section:

![astr](https://user-images.githubusercontent.com/40228854/49289773-2a399c80-f4a5-11e8-8d86-3cc80efdd46f.JPG)

@helenemccarron There is still problem with University Physics. I have baked this book, clear the cache from server and from browser, but additional header appears.

![up_before](https://user-images.githubusercontent.com/40228854/49289856-653bd000-f4a5-11e8-8e73-153a1b6e6e99.JPG)

It should be removed. This is how it look in PDF:

![up_pdf](https://user-images.githubusercontent.com/40228854/49289876-77b60980-f4a5-11e8-82e1-2149d0f28718.JPG)
